openslides_backend/action/actions/mediafile/upload.py

```diff
@@ -31,23 +28,19 @@
 
 
 @register_action("mediafile.upload")
-class MediafileUploadAction(
-    CreateActionWithDependencies,
-    CreateActionWithListOfSpeakersMixin,
-):
+class MediafileUploadAction(MediafilePermissionMixin, CreateAction):
     """
     Action to upload a mediafile.
     """
 
     model = Mediafile()
     schema = DefaultSchema(Mediafile()).get_create_schema(
-        required_properties=["title", "meeting_id", "filename"],
+        required_properties=["title", "owner_id", "filename"],
         optional_properties=["access_group_ids", "parent_id"],
         additional_required_fields={"file": {"type": "string"}},
     )
     permission = Permissions.Mediafile.CAN_MANAGE
-
-    dependencies = [ListOfSpeakersCreate]
+    skip_archived_meeting_check = True
 
     def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Here, the code from the old `resource.upload` has to be included: if `token` is given, `pop` the `filename` from the instance. Also the `get_updated_instance` method from the old action has to be included.

Reply:
Merge the `resource.upload` action into the `mediafile.upload` action.